jcli/src/jcli_app/vote/committee/communication_key.rs

```diff
@@ -9,6 +10,9 @@ use std::{
 };
 use structopt::StructOpt;
 
+pub const COMMUNICATION_SK_HRP: &str = "p256k1_communicationsk";
+pub const COMMUNICATION_PK_HRP: &str = "p256k1_communicationpk";
```

Comment:
Let's make these prefixes shorter, maybe `p256k1_vcommpk`/`p256k1_vcommsk` (as in voting committee/communication).